.meshrc.yaml

```diff
@@ -27,6 +27,20 @@ sources:
       jsonSchema:
         endpoint: ${API_URL}/
         operations:
+          - type: Query
+            field: adminSamples
+            path: /samples/index_v2.json
```

Comment:
the `path` field here and below is arbitrary -- we are not hitting the rails backend at all. It complains if I don't have these, though. Can we put a placeholder string there?

Reply:
I don't think it hurts anything to put a place holder in there. I am not sure why path is required but I do know that if you put vars directly in the path the query expects the args differently (outside of the input:{}). Any other thoughts?

Reply:
could we add these as `additionalTypeDefs` instead of under the Rails source?  There's a couple of existing ones for our version stuff: https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/62cf994df7bdfa9af71c76cb6bf2dc2ec8f0c923/.meshrc.yaml#L300-L307

Then you can add a resolver for those types that calls whatever you need it to.

Reply:
here's the resolver for the custom additional type above:
https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/62cf994df7bdfa9af71c76cb6bf2dc2ec8f0c923/resolvers.ts#L1140-L1143

Reply:
passing on this for now as these resolvers are temporary, per offline discussion